appcenter-data/Data.js

```diff
@@ -50,10 +52,43 @@ function read(documentId, partition, readOptions) {
  * If the operation fails, the promise is rejected with an exception containing the details of the error.
  */
 function create(documentId, document, partition, writeOptions) {
-    if (writeOptions === undefined) {
+    if (!writeOptions) {
```

Comment:
Please do not revert that as it would match 0 or false.
On typescript we will be more strict in the definition also.
null is pass through to native SDK which allows it, so it's actually better to check for undefined.

Reply:
Done